src/infrastructure/repositories/category.repository.ts

```diff
@@ -7,6 +7,7 @@ import {
   UpdateCategoryDto,
   AddCategoryDto,
 } from '@/presentation/category/category.dto';
+import { Context } from '@test/mock/prisma/prisma-service.mock';
```

Comment:
I think it's not necessary here, or it is?

Reply:
is not :)